### ISSUE TITLE
please add nvidia/Nemotron-4-340B-Instruct

### ISSUE THREAD

**Issue by gileneusz** [ollama/ollama#5068]
my GPUs are not utilized fully, I need to spin my H200s!!

just kidding, need quantized version of the model ;)

**Comment by Twisig** [comment]
How to run this model on macOS ? #nemotron

**Comment by Nehc** [comment]
the quantized version, theoretically, will require 170 GB of RAM, right? In general, this is not so hopeless! )

**Comment by sujumayas** [comment]
I am tempted to buy a new mac...

**Comment by kozuch** [comment]
llama3:70b-instruct-q2_K (26GB in size, cca 2.9bpw) actually does not perform that bad. On Nemotron we could probably test how agresive quantization behaves with large models. 2.9bpw for Nemotron would make about 126 GB. Can still be run for few $s on rented HW. Would be really interesting to test.

**Comment by barclaybrown** [comment]
Agree. In some applications, speed doesn't matter too much. Often I'm building applications that iterate through a dataset and use an LLM to do something on each item. Whether these take a few minutes or a few hours doesn't really matter, so I usually develop and test with a small, fast model, but then run the final with a bigger, smarter model. Would love to have Nemotron as an option, even it it runs slowly.

**Comment by Nehc** [comment]
there is already [guff](https://huggingface.co/mradermacher/Nemotron-4-340B-Instruct-hf-GGUF), I even uploaded it to ollama, but it says that the Nemotron model is not supported...

**Comment by gileneusz** [comment]
It's still a good model and can be included for testing purposes. Additionally, it has a very favorable license for dataset generation. However, it's not added because most people don't have access to DGX platforms, so they can't really use it...

**Comment by Nehc** [comment]
> Теперь, когда у нас есть модель Llama 3.1 405B, я думаю, что работа с Nemotron-340 начинает немного устаревать. Конечно, будет интересно, хотя приоритет для меня снизился из-за новой Llama.

I understand, yes. However, I had the opportunity to compare these models on some tasks, and I can say for sure that at least on some (perhaps a little specific to my tasks) Nemotron is directly noticeably cooler than the LLama 405b

**Comment by wottpal** [comment]
Did someone manage to get the Nemotron model running on a Mac locally? Would it even work let's say on a fully-speced M3 Max?
